lib/blocks/blockreader.go

```diff
@@ -57,9 +58,8 @@ func IsStartLine(line string) bool {
 }
 
 func IsFinishLine(line string) bool {
-	if line == "`)\n" { // acctest
-		return true
-	} else if strings.HasPrefix(line, "`,") { // acctest
+	spaceLeftAccTestMatcher := regexp.MustCompile("^[[:space:]]*`(,|\\)\n)")
```

Comment:
Could we make this a constant somewhere?

Reply:
Done.